extension/fuzzySearch.js

```diff
@@ -1,29 +1,19 @@
 // Put all the javascript code here, that you want to execute after page load.
-document.addEventListener("keydown", addExtension);
-document.addEventListener("mousedown", addExtension);
+addExtension();
```

Comment:
This here seems to cause the problem. The search bars have not loaded in yet when this is called, so the querysSelectors don't find them.